contrib/pureconfig/jvm/src/main/scala/eu/timepit/refined/pureconfig/error/PredicateFailedException.scala

```diff
@@ -0,0 +1,4 @@
+package eu.timepit.refined.pureconfig.error
+
+final case class PredicateFailedException(message: String)
```

Comment:
Adding it to the core module sounds good, should be no reason against having it there! 👍